src/azure-cli/azure/cli/command_modules/acr/_archive_utils.py

```diff
@@ -156,6 +166,8 @@ def _load_dockerignore_file(source_location):
         encoding = "utf-8-sig"
 
     ignore_list = []
+    if docker_ignore_file == docker_ignore_file_override:
+        ignore_list.append(IgnoreRule(".dockerignore"))
```

Comment:
Hi @juliusl, thanks for your comments. I had to exclude the `.dockerignore` file in case it's overridden as it gets picked up by the registry build task. Without doing this instead of overrding `.dockerignore` I would be merging the 2 dockerignore files.